### PR TITLE
Closes #387: Add stylesheet reference for CKEditor

### LINKED ISSUE #387: Apply Bootstrap styles inside CKEditor
## Motivation
Currently, styles can be applied, but can only be seen once the page is saved, which makes for a poor editing experience.

### Describe the solution you'd like
When styles are selected, they should be applied to the text in the editor, so the resulting styling can be seen in real time while edits are being made.

### Roles and Permissions considerations
A clear and concise description of how each of the following roles would be impacted by this change:
This would apply to all roles that can make edits with CKEditor.

### PR DESCRIPTION
**Pull request by danahertzberg**
## Description
Goal: see the styles being applied while editing in the CKEditor.

Thank you @trackleft for the solution!

## Related Issue
Closes #387

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I've added this Pull Request to the AZ Quickstart Github project and set it to "Review in progress".
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by camikazegreen — DISMISSED** [review]

**Comment by joeparsons** [comment]
We might be able to use our local copy of AZ Bootstrap via this hook:
https://api.drupal.org/api/drupal/core%21modules%21ckeditor%21ckeditor.api.php/function/hook_ckeditor_css_alter/9.1.x

**Comment by danahertzberg** [comment]
> We might be able to use our local copy of AZ Bootstrap via this hook:
> https://api.drupal.org/api/drupal/core%21modules%21ckeditor%21ckeditor.api.php/function/hook_ckeditor_css_alter/9.1.x

I was wondering about that option. I'll probably need some help with the hook implementation, but agree this is a better long term solution.

File: themes/custom/az_barrio/az_barrio.info.yml [hunk 1]
```diff
@@ -36,3 +36,6 @@ regions:
   footer: Footer
   footer_sub: 'Footer sub'
   page_bottom: 'Page bottom'
+
+ckeditor_stylesheets:
+  - https://cdn.digital.arizona.edu/lib/arizona-bootstrap/main/css/arizona-bootstrap.min.css
```
Comment (danahertzberg) [inline themes/custom/az_barrio/az_barrio.info.yml:41]:
```suggestion
  - https://cdn.digital.arizona.edu/lib/arizona-bootstrap/latest/css/arizona-bootstrap.min.css
```

**Comment by danahertzberg** [comment]
Will create follow up ticket to have this match site's version of Arizona bootstrap

**Review by camikazegreen — APPROVED** [review]

**Review by mmunro-ltrr — APPROVED** [review]

**Review by joeparsons — APPROVED** [review]

**Review by trackleft — APPROVED** [review]

**Comment by joeparsons** [comment]
No idea why the tests passed after triggering a Probo rebuild a third time.  A similar PHPunit failure is also showing up on @kevcooper's PR: #763.

I was unable to reproduce any of the errors/failures locally.

**Review by kevcooper — APPROVED** [review]

**Pull request merged**